appcelerator/titanium_mobile
# [TIMOB-25258] Android: Fixed bug where ListView/TableView ignores Ti.UI.SIZE for height. (Introduced in 6.2.0.)

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25258

**Notes:**

- This bug was introduced in Titanium 6.2.0 when the RefreshControl feature was added to Android.
- Issue was caused by Google's "SwipeRefreshLayout" class, which ignores `WRAP_CONTENT` for height. Modified derived class to support it.

**Test:**

1. Run the project attached to [TIMOB-25258](https://jira.appcelerator.org/browse/TIMOB-25258).
2. Verify that the displayed table matches ticket's "expected_good.png" screenshot.
